### ISSUE TITLE
'ready' value is incorrect for some number of ticks

### ISSUE THREAD

**Issue by C-Higgins** [plaid/react-plaid-link#163]
```js
  return {
    error,
    ready: !loading || iframeLoaded,
    exit: plaid ? plaid.exit : noop,
    open: plaid ? plaid.open : noop,
  };
```

`loading` refers to the script being loaded, which always happens before `plaid` can be initialized. 
The order of events is
1. the hook mounts; loading is true; returns `ready:false`; effect hook runs but returns early because loading is true; 
2. load finishes; function reruns with loading==false; **returns `ready:true`, then** effect hook runs again, sets plaid

This means that `ready` becomes true while `plaid` is still null. You cannot rely on the `ready` prop to be correct. I think you probably meant to write     `ready: !loading && plaid,`

**Comment by skylarmb** [comment]
@C-Higgins yep, can verify this is an issue. Thanks!

**Comment by skylarmb** [comment]
this was fixed in 3.1.0
